apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -57,35 +51,30 @@ const Sidebar: React.FC<Properties> = ({ menuItems }: Properties) => {
 				</Link>
 				<nav className={styles["menu"]}>
 					{menuItems.map(({ href, icon, label }) => (
-						<>
-							<Button
-								className={styles["menu-item"]}
-								href={href}
-								iconName={icon}
-								key={label}
-								label={label}
-							/>
+						<div key={label}>
 							<Link
+								activeClassName={styles["active"]}
```

Comment:
```suggestion
							<Link
							  key={label}
								activeClassName={styles["active"]}
```
I think you should omit this div and pass key directly into the Link component